packs/one-shot-bestiary/dinner-at-lionlodge/auldegrund-grimcarver.json

```diff
@@ -161,7 +161,7 @@
                     },
                     "secondary": {
                         "casters": 3,
-                        "checks": "Arcana, Nature, Occultism,  Religion, or Society (whichever is required to Recall Knowledge about the creature being duplicated); Crafting; Deception"
+                        "checks": "Arcana, Nature, Occultism, Religion, or Society (whichever is required to Recall Knowledge about the creature being duplicated); Crafting; Deception"
```

Comment:
Line 160 is also missing the closing parentheses. "Arcana or Occultism (master, the check has the secret trait"